SwiftVoiceCallingApp/SwiftVoiceCallingApp/Controller/PlivoCallController.swift

```diff
@@ -162,15 +162,28 @@ class PlivoCallController: UIViewController, CXProviderDelegate, CXCallObserverD
             UtilClass.makeToast(error.localizedDescription)
             UtilClass.hideToastActivity()
             print(error.localizedDescription)
-//            UtilClass.setUserAuthenticationStatus(false)
-//            UserDefaults.standard.removeObject(forKey: kUSERNAME)
-//            UserDefaults.standard.removeObject(forKey: kPASSWORD)
-//            UserDefaults.standard.synchronize()
-//            let _mainStoryboard = UIStoryboard(name: "Main", bundle: nil)
-//            let loginVC: LoginViewController? = _mainStoryboard.instantiateViewController(withIdentifier: "LoginViewController") as? LoginViewController
-//            Phone.sharedInstance.setDelegate(loginVC!)
-//            let _appDelegate: AppDelegate? = (UIApplication.shared.delegate as? AppDelegate)
-//            _appDelegate?.window?.rootViewController = loginVC
+            
+            switch (error! as NSError).code {
+            case 502:
```

Comment:
If we are sending this from SDK why don't we just stop this in our SDK itself.

Reply:
If this is a simple change on our SDK we should do this first then upgrade to 2.8